raiden-dapp/src/components/AddressInput.vue

```diff
@@ -146,13 +161,37 @@ export default class AddressInput extends Mixins(BlockieMixin) {
       this.errorMessages.push(this.$t(
         'address-input.error.no-checksum'
       ) as string);
-    } else if (AddressUtils.checkAddressChecksum(value)) {
-      this.input(value);
     } else if (
       !AddressUtils.isAddressLike(value) &&
       AddressUtils.isDomain(value)
     ) {
       this.resolveEnsAddress(value);
+    } else if (
+      AddressUtils.checkAddressChecksum(value) &&
+      !Object.keys(this.presences).includes(value)
```

Comment:
```suggestion
      !(value in this.presences)
```
Nitpick: more performant, as doesn't require creating an array